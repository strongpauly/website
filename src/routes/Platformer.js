import React, { Component } from "react";
import { HiFrame } from "../components/HiFrame";

export class Platformer extends Component {
  render() {
    return (
      <div id="platformerContainer">
        <p>
          A work in progress writing a platform game using react. It&apos;s
          source can be found{" "}
          <a href="https://github.com/strongpauly/platformer">here</a>.
        </p>
        <HiFrame
          src="https://platformer.potsides.com/platformer/index.html"
          title="platformer"
        ></HiFrame>
      </div>
    );
  }
}
