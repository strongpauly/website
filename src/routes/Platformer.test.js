import React from 'react';
import {Platformer} from './Platformer';
import {shallow} from 'enzyme';

/* eslint-env jest */

describe('Platformer', () => {
  it('renders without crashing', () => {
    const app = shallow(<Platformer/>);
    expect(app).toMatchSnapshot();
  });

  it('creates a #minesweeperContainer', () => {
    const app = shallow(<Platformer/>);
    expect(app.find('#platformerContainer')).toHaveLength(1);
  });
});
