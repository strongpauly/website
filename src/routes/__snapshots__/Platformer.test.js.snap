// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Platformer renders without crashing 1`] = `
<div
  id="platformerContainer"
>
  <p>
    A work in progress writing a platform game using react. It's source can be found
     
    <a
      href="https://github.com/strongpauly/platformer"
    >
      here
    </a>
    .
  </p>
  <HiFrame
    src="https://platformer.potsides.com/platformer/index.html"
    title="platformer"
  />
</div>
`;
